### DISCUSSION TITLE
Feature Request: Copy to Clipboard Functionality in Code Mockup Component

### DISCUSSION THREAD

**Discussion by 0xmuneeb** [saadeghi/daisyui#1743] [Ideas / feature requests]
### **Hello, the amazing Team of Daisy UI,**

I get to know about this wonderful _Daisy UI_ which is, I must say, the missing components library for tailwindcss. And the best part is that it is totally **open-source & FREE**. 
I am using code mockup component and it seems like there is no functionality or any button to copy the block content. 
So I would like to request the addition of a **"Copy to Clipboard"** button in the Code mockup component. And obviously this would be a choice for the developers to include this modifier if they want to use this functionality. 

Best regards,
_Muneeb Ur Rehman_

**Comment by saadeghi** [comment]
You need a JS code for copying to the clipboard. daisyUI is just a CSS file so we can't really do that.  
Depending on the JS framework you're using, there are npm packages which can do this.

Let me know if you have any questions

**Reply by 0xmuneeb** [reply]
Thanks for your response. I meant to say that there should be an option to include the copy button at the top right or top left of the code mockup. I tried manually to add it but it doesn't stick at top.
